recipe/meta.yaml

```diff
@@ -1,39 +1,44 @@
-{% set version = "1.3.2" %}
+{% set name = "blosc" %}
```

Comment:
When you install one wants to run it you run, `import blosc` I thought this would be less confusing, but the repo is called `python-blosc`. Changing the name back to python-blosc
e0f075475ca69043d9b417db241020246d3dfd3e